src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java

```diff
@@ -59,21 +59,21 @@
   private static final String ATTRIBUTES_SUPPORTED_GRADLE_VERSION = "4.0";
 
   public Set<ResolvedDependency> findResolvedDependencies(Project rootProject, boolean allConfigurations) {
-    Configuration copyConfiguration = createCopyConfiguration(rootProject);
-
-    return rootProject.getAllprojects().stream().flatMap(project -> project.getConfigurations().stream())
+    rootProject.getAllprojects().forEach(p -> p.getConfigurations().forEach(c -> {
+      System.out.println(c.getAllArtifacts().size() + ": " + c.getName());
+    }));
```

Comment:
Minor. You can make this an expression lambda by dropping the curly braces "{}"

Reply:
> Minor. You can make this an expression lambda by dropping the curly braces "{}"

Also you can avoid the nested forEach statements by using flatMap e.g.
```
rootProject.getAllprojects().stream()
        .flatMap(p -> p.getConfigurations().stream())
        .forEach(c -> System.out.println(c.getAllArtifacts().size() + ": " + c.getName()));
```

Reply:
> Minor. You can make this an expression lambda by dropping the curly braces "{}"

This was just for local tests during development and should be merged 😅
Removed: 49f3eb6e60a343ed61f8f69bc2f937f68da4995d